benchmark/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/zsh
+
+ROOT=$PWD/..
+rm -rf runner/out runner/processor-1.0-SNAPSHOT.jar runner/com tachiyomi runner/lib
+./gradlew :jar
+cd $ROOT/benchmark/exhaustive-processor
+./gradlew build
+cp processor/build/libs/processor-1.0-SNAPSHOT.jar ../runner/
+cd $ROOT/benchmark
```

Comment:
Should we create a temporary directory for this instead of checking out tachiyomi in our current checkout?